pycontw/pycon.tw
# Update overview style

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies


## Description
**Describe what the change is**
You don't need to explain HOW you change the code. Your code, including comments in the code should explicitly
self-describe enough about how you change the code.
* Fix underscore for overview page submenu button
* Update overview page header style
* Also update the format of html followed by black formatter (Not sure if it is a better revision, I can revert this if needed)
## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to home page
2. Click the overview button in menu
3. Check the overview page

## Expected behavior
A clear and concise description of what you expected to happen.
The style will show as following images
![image](https://user-images.githubusercontent.com/18432820/89568153-74b41480-d855-11ea-995a-894673296c39.png)
![image](https://user-images.githubusercontent.com/18432820/89568181-82699a00-d855-11ea-8c19-c0712f167df7.png)

## Related Issue
If applicable, refernce to the issue related to this pull request.
#841 #856 
## More Information
**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to wirte the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)
